bin/python/coverage_merge_script.py

```diff
@@ -0,0 +1,56 @@
+__author__ = 'Jan Hybs'
```

Comment:
Add comment to beginning of each script with brief description of its purpose and usage example.